make_test_images/src/make_test_images.rs

```diff
@@ -67,7 +68,7 @@ pub struct Config {
     /// The signing algorithm to use
     pub alg: String,
     /// A url to a time authority if desired
-    pub ta: Option<String>,
+    pub tsa: Option<String>,
```

Comment:
We decided somewhere that we are using ta instead of tsa, and for this value, I used ta_url in c2pa_tool, so it should be changed to ta_url here as well.

Reply:
I left that to you upcoming PR.  Do you want me to change it?

Reply:
I thought you were going to change tsa to ta in the config file, but if we are going to rename the field, I'd rather rename it once.

Reply:
changed to tsa_url consistently with most of the source code.